.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,4 +6,4 @@ Delete them if they are not appropriate for this pull request.
 - [ ] I have added suitable tests demonstrating a fixed bug or new/changed feature to the test suite in `tests/`
 - [ ] I have properly documented new or changed features in the documentation or in the docstrings
 - [ ] I have properly explained unusual or unexpected code in the comments around it
-- [ ] I have formatted my code using `black -t py36` 
\ No newline at end of file
+- [ ] I have ran pre-commit hooks while doing the commit or running `pre-commit run --all-files`
```

Comment:
And maybe this should be changed to something like "I have commit hooks installed with `pre-commit install`"?

Reply:
To the stake with it!